semantic.json

```diff
@@ -0,0 +1,58 @@
+{
+  "version": "2.8.3",
+  "base": "node_modules/fomantic-ui",
+  "paths": {
+    "source": {
+      "config": "src/theme.config",
+      "definitions": "src/definitions/",
+      "site": "src/site/",
+      "themes": "src/themes/"
+    },
+    "output": {
+      "packaged": "../../public/fomantic/",
+      "uncompressed": "../../public/fomantic/components/",
+      "compressed": "../../public/fomantic/components/",
+      "themes": "../../public/fomantic/themes/"
+    },
+    "clean": "../../public/fomantic/"
+  },
+  "permission": false,
+  "autoInstall": false,
+  "rtl": false,
+  "admin": false,
+  "components": [
+    "accordion",
+    "breadcrumb",
+    "button",
+    "checkbox",
+    "comment",
+    "container",
+    "divider",
```

Comment:
```suggestion
    "dimmer",
    "divider",
```

Reply:
Fixes `Modal: UI Dimmer, a required component is not included in this page` seen on page user settings -> security.